### ISSUE TITLE
Meta feedback

### ISSUE THREAD

**Issue by jackyho112** [UBC-MDS/DSCI_522_Group405#37]
Discuss the metric choice

Discuss the model choice - linear regression

Outlier plot

Mention test and train error

Discuss overfitting perhaps

Discuss complexity

Criterion is mean squared error for Random Forest but outputting median and using MAE
use Randomized Search instead of grid search so we can try more hyperparameter values
